3ds Max/Max2Babylon/BabylonSkipFlatten.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Windows.Forms;
+using Autodesk.Max;
+using ActionItem = Autodesk.Max.Plugins.ActionItem;
+
+namespace Max2Babylon
+{
+    class BabylonInvertSkipFlatten:ActionItem
```

Comment:
`class BabylonSkipFlattenFlagToggle : ActionItem`

would improve the spacing and more clear naming schema